obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -30,6 +30,10 @@ BuildRequires:  libelf-devel
 BuildRequires: fdupes
 %endif
 
+%if 0%{?centos_version}
+BuildRequires:  elfutils-libelf-devel
```

Comment:
Won't this change affect Centos 7 and 8? If so, why didn't we need this for 7 before? Same comment for `python2` below.